core/scss/components/site-search/_site-search.scss

```diff
@@ -0,0 +1,73 @@
+@charset "UTF-8";
+
+//
+// Site Search
+//
+// A global site search component
+//
+// Markup: ../templates/components/site-search/site-search.twig
+//
+// Style guide: Components.Site Search
+//
+.su-site-search__input {
+  @include font-smoothing();
+  @include padding(.3em 1em);
+
+  display: inline-block;
+  height: 40px;
+  max-width: 100%;
+  border-radius: 20px;
+  font-size: .8em;
+
+  &::placeholder {
+    color: $color-black;
+    opacity: 1; // For Firefox
+  }
+
+  @media #{$breakpoint-lg} {
+    width: 320px;
```

Comment:
Would `max-width` be better here?
Would the value be better specified in (r)em's?

Reply:
Yes, I'm changing it to max-width and using 20em.